### ISSUE TITLE
Q3 2022 ethereum.org product roadmap

### ISSUE THREAD

**Issue by samajammin** [ethereum/ethereum-org-website#7166]
*To make our work more accessible and to foster more community collaboration, our team publishes an overview of our quarterly roadmap goals. See last quarter's roadmap here: #6161.*

**Greetings fellow Ethereans!**

[Our vision with ethereum.org](https://ethereum.org/en/about/) continues to be to create the best portal for Ethereum's growing community. We’re an educational resource focused on helping onboard millions of visitors to Ethereum each month. Here's a look at our major Q3 initiatives to advance this vision.

Quick point of clarification: ***this is the roadmap for [ethereum.org](http://ethereum.org/) the website, not Ethereum the protocol***. If you’re looking for the Ethereum roadmap, check out information on [upcoming network upgrades](https://ethereum.org/en/upgrades/) and learn more about [Ethereum governance](https://ethereum.org/en/governance/).

With that out of the way, let’s dive in.

# The must-haves

*The epics our core team is committing to shipping this quarter.*

## 🔀 Post-Merge prep

Core Ethereum developers are in the final stages of testing and preparing for [The Merge](https://ethereum.org/en/upgrades/merge/). Last quarter we shipped updates across [ethereum.org](http://ethereum.org) (#6281) and the [Ethereum Staking Launchpad](https://launchpad.ethereum.org/en/) (https://github.com/ethereum/staking-launchpad/pull/493) to help prepare our community for this Ethereum protocol upgrade. These updates handled all pre-merge updates: content changes that can be made **_before_** The Merge takes place.

This quarter we’re turning our focus to post-merge content updates: content changes that should be rolled out **_after_** Ethereum completes The Merge successfully. In a similar fashion, we’ll run a [page-by-page audit](https://docs.google.com/spreadsheets/d/1Nisi3_FY8nwOlLwPsTPm0jlgtAhypH0-pjatGDMSz7M/edit#gid=0) of ethereum.org and the Ethereum Staking Launchpad and rollout updates across both sites. We’ll also work to identify and fill gaps in site content - topics we should cover in a post-Merge world.

Want to get involved? Join the conversation in our [#merge-prep Discord channel](https://discord.gg/aEgUGmEHUT).

## 🌏 Translate the EF blog

We'll finish the site rebuild we started in Q2 and deploy a fresh blog codebase with internationalization support this quarter.

The blog repo is currently private, so not much to share at this point, but we're hoping to open source the codebase as part of the launch. In the meantime, you can view our [design updates](https://www.figma.com/file/PmoL6wNs8ck8H3OuUvUItj/EF-blog?node-id=0%3A1) and test out the [latest preview deploys](https://thunderous-stroopwafel-2488b4.netlify.app/). Feedback is always welcome - feel free to comment directly in Figma or join the conversation in Discord.

After launch, we'll publish a blog post detailing how to get involved with the community translations, so stay tuned!

## 🎨 Open design system

The open design system is and will continue to be an ongoing effort that we kicked off in Q2. The design system will help establish and document our design language, which anyone in the community can use to create reusable resources. The aim here is to increase our UI's consistency and productivity for our design and development community.

In Q2, we mapped and divided our roadmap for the design system into 1) foundations, 2) components, and 3) layouts. With the foundations in place, in Q3, we’ll focus on building reusable components that help make layouts.

These components will be divided into two buckets:
1. Base components: small elements that can be used in different positions and can be forms, buttons, navigation, and cards.
2. Composed components: combinations of various base components that form the core of layouts, including header, footer, and page hero.

By the end of the quarter, we will release a new update to the design system ready with some base layouts made out of combinations of components. We’re aiming to have a base layout, a markdown layout, a use case page layout, and the homepage.

Want to get involved? [Follow along in Figma ](https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System) and join the conversation in our [#design Discord channel](https://discord.gg/bKycYhVUwV).

## 🖥 Implement UI library

Another Q2 carryover initiative (#6374) closely tied to the design system - to improve our code quality, improve productivity, website accessibility and overall UX, we’re adding a UI library to our codebase.

We’ll kick off the implementation work this quarter and start rolling out [Chakra UI](https://chakra-ui.com/) components across the codebase. This is a significant undertaking, which will likely involve large refactors to clean up duplicate code and to integrate the new design system that we expect to be working on into Q4.

Want to get involved? Join the conversation in [our #website-code Discord channel](https://discord.gg/7WKTFRARe5).

## 🦔 Build a new Geth website

The current Geth site ([geth.ethereum.org](https://geth.ethereum.org/)) has served its purpose for a long while but is far from perfect - it contains some out-of-date and incomplete information, lacks modern website features like built-in search and the overall aesthetics lack a modern look that undersells what Geth has become - a core piece of Ethereum infrastructure and public good for the ecosystem. 

This quarter we’ll kick off design, content, and code to build a new Geth site. We plan to provide expanded educational information, accurate and robust documentation, and a modern design fitting for such a crucial resource in the space.

## 💖 Implement HEART framework

The HEART framework is a well-established, proven, UX-focused methodology to improve software products. We plan to apply the HEART framework to different user groups and aspects of our products, starting in Q3 with the translation program.

We’ll implement the HEART framework across all aspects of this contributor funnel - Happiness, Engagement, Adoption, Retention, and Task success. We’ll pull data, surface insights, and identify key takeaways that will guide our operation and product priorities for the translation program next quarter. With the translation program as a testing ground, we’ll also consider applying the HEART framework more broadly across all our contributor and end-user funnels.

Want to get involved? Join the conversation in [our #translate Discord channel](https://discord.gg/bTCfS8C).

## 💰 Wallet upgrades

In Q2, we revamped our [Find Wallet page](https://ethereum.org/wallets/find-wallet) (#6274).

While this was a large release that introduced substantial enhancements to the data gathered on wallets and how users can filter and find an Ethereum wallet, there are still remaining features and cleanup items that we identified to improve the find wallet page. This quarter we’ll continue introducing additional improvements that were scoped out of the initial release.

Want to get involved? Join the conversation in our [#wallets Discord channel](https://discord.gg/aQEq7GQsgE).

## 💯 Learning quizzes

Many aspects of Ethereum are complex and can be difficult to grasp. A great way to ensure important content is understood is by immediately testing knowledge after it is presented. We’ll introduce optional quizzes across [ethereum.org](http://ethereum.org/) pages to ensure that readers grasp the essential concepts to allow them to be meaningful participants in the Ethereum ecosystem. Along with our feedback surveys, we expect these quizzes to drive insights into how to improve our content resources further.

Over the long term, we envision this will set the foundation for more structured, sequential learning resources (perhaps with web3 gamification) on ethereum.org.

## 🗣️ User insights

Last quarter we shipped a new feedback widget (#6252) integrated with custom surveys across all pages on [ethereum.org](http://ethereum.org) and started collecting data. This quarter we will focus on validating and analyzing your responses and formulating useful insights to improve our site. We’ll build reports and consider how we might publish research findings as a public good to the Ethereum product community. We will also use this data to influence our content strategy decisions and priorities.

Want to get involved? Join the conversation in our [#design Discord channel](https://discord.gg/bKycYhVUwV).

## 🌍 Revamp translation program resources

The [Translation Program page](https://ethereum.org/en/contributing/translation-program/) is dedicated to the [ethereum.org](http://ethereum.org) Translation Program and contains key information on what the program is, how to get involved, how to succeed as a translator and even guides on building out your own translation initiative. 

It’s been constantly growing in size and value but lacks structure. We’ll review the information architecture and user flow on the page, break out some sections into dedicated subpages, create subpages for other web properties that we are translating (specifically the [Staking Launchpad](https://launchpad.ethereum.org/en/) and the [EF blog](https://blog.ethereum.org/)), and refresh the information on the page.

Want to get involved? Join the conversation in [our #translate Discord channel](https://discord.gg/bTCfS8C).

# The nice-to-haves

*These epics are up next on our priority list. We’re not committing to shipping anything this quarter, but we plan to make progress on these and want to share these publicly so the community can give input and get involved!*

## 🏫 Ethereum for educators

We’re narrowing in on how to best build resources for educators looking to teach about Ethereum. In Q2, we collected a lot of resources for educators - like university courses, case studies, and open access academic papers - and actively engaged with academics for their feedback, ideas, and input. This quarter we hope to pick up the work to design and build a page to offer these free resources to teachers and learners across the globe.

Want to get involved? Join the conversation on #6283 or in our [#content Discord channel](https://discord.gg/nZsD6hTxff).

## 📖 Content strategy

Thanks to the incredible contributions from hundreds of community members, we’ve created a large swath of excellent educational resources on ethereum.org. However, we still have gaps in our content where we are not addressing some fundamental questions about how Ethereum works. Meanwhile great new content ideas constantly surface in our Discord and on GitHub but we don’t yet have a solid framework on how to prioritize these within our content roadmap.

Through SEO tools and manual content auditing, we’re aimig to identify such gaps and build an editorial calendar that helps prioritize content opportunities by their impact, allowing us to create more critical content faster.

Want to get involved? Join the conversation in our [#content Discord channel](https://discord.gg/nZsD6hTxff).

## 🪜 Restructure documentation

With the help of a community of technical content contributors, our developer documentation resources have exploded to nearly 100 pages. These resources live within a site structure that was originally targeted at web3 dapp developers yet over time, we’ve received feedback that many resources are useful to a broad range of visitors - hobbyists, designers, node operators, end-users, economists, entrepreneurs, and more user types beyond pure “developers”.

This quarter we hope to rethink the overall information architecture of our documentation to best support a broader base of users and to ensure we surface key information that is often buried deep within technical topics.

Want to get involved? Join the conversation in our [#content Discord channel](https://discord.gg/nZsD6hTxff).

## 💰 Wallet resources

Through our recent work on our new Find Wallet page (#6274), we surfaced opportunities to improve and expand the educational resources on the world of Ethereum wallets, which are critical onboarding products to all Ethereum users.

We’ll work to expand our content around wallets, such as explainers on self-custody and how keys work, resources on the different kinds of accounts (EOAs vs. contracts), and the respective wallet interfaces they can use (private-key wallet, smart contract wallet, and MPC wallets).

Want to get involved? Join the conversation in our [#content Discord channel](https://discord.gg/nZsD6hTxff).

## 🎡 JSON-RPC playground

The [JSON-RPC API](https://ethereum.org/en/developers/docs/apis/json-rpc/) is a crucial list of endpoints provided by all Ethereum clients yet is not widely or consistently documented. Several folks have been working valiantly to improve the state of affairs, building out new resources such as the [specification](https://github.com/ethereum/execution-apis/) and the [interactive playground](https://ethereum.github.io/execution-apis/api-documentation/).

These resources are a great start, and we’d like to help push things forward. We’re hoping to work with @lightclient and other community members to improve the JSON-RPC playground, allowing developers to find, learn about, and test various endpoints to build products for end-users. We’re hoping these efforts will improve the lives of client teams, dapp and infrastructure developers, and end-users across the Ethereum ecosystem.

Want to get involved? Join the conversation in the [#json-rpc-api channel of the Eth R&D Discord](https://discord.gg/NJW4BJe3XZ).

# How does that sound?

We always appreciate feedback on our roadmap - if there's something you think we should work on, please let us know! We welcome ideas and PRs from anyone in the community.

Get involved: [Learn more about contributing](https://ethereum.org/en/contributing/), [hit us up on Twitter](https://twitter.com/ethdotorg), or join the community discussions in [our Discord](https://discord.gg/CetY6Y4).

**Comment by vdusart** [comment]
Hey @samajammin, the `Find Wallet page` link in the `Wallet upgrades` is not working :slightly_frowning_face:

**Comment by samajammin** [comment]
**Translation KPIs**

- Reduce the number of QA issues in Crowdin by 20%
- Update 200 content buckets between all languages
- Add translations for Essential pages in at least five new languages
- Add 5 new language leads

**Community KPIs**

- Allow 2 > PRs open longer than one month
- Hit 15 GitHub power contributors (at least 3 PRs)
- Add 5 new Discord guides
- Host 3 community calls (monthly public calls)
- Host 3 MVP hangouts (monthly get-togethers for language leads, Discord guides & GitHub power contributors)

**Comment by pilvijo** [comment]
Thanks for the great community call few days ago. Lots of work to do during Q3 2022. 

And very clear presentation above: me like :)

Once again, I worry a bit about the information share about the Merge: Ethereum as is --> to be -style.
Could wallet providers help with the information share, if community didn't have any effective channel to serve external users. For me, wallet-provider has been one of the most important link to understand Ethereum and keep up to date what is happening.

The other question: does it make sense to continue ethereum.org - translations before Merge. For me all info is good info, even history, but is this work useless at moment for the community.

**Comment by Moesaawi** [comment]
> Hey @samajammin, the `Find Wallet page` link in the `Wallet upgrades` is not working 🙁 I think you just forgot the /en in the middle

Hi dear,

It depends on where you earned your Ethereum, further explanation you can find below which is fully useful. The wallet should be obvious like going to your profile. If not we would recommend to contact customer support, and check if the website, application, or any type of system granted you with our **smart contract**. 

please note our definition of the word " Ethereum Wallet " , it simply means exchanging any type of currency, crypto-currency, NFT's, and so on, A lot can be explained here as of activities used within our block-chain is financially booked. As there is an ERP system ( Enterprise Resource System ) Which is used in terms of financial data, and records, at the same time keeping full privacy of the transactions. Transferring the accountability to the intention of the final person transferring the money to his account, according to his own countries and policies, which some has banned the usage of Crypto-Currency. 

We simply allow you to follow your true intentions, desires, dreams , and more by buying our smart contracts. Allowing you  to exchange any type of item you personally believe with real value. 



How to create and find your wallet?

It is a kind of serial number example A wallet can be found anywhere depending on the activity of where your platform granted you with Ethereum, as i previously mentioned it is a smart contract, therefore, the whole world is contributing to it at rapid pace,. i will give below examples of places you could have earned it. Please note we highly recommend that you avoid unknown platforms, random people from the internet, asking you for anything related to cryptocurrency wallets block chains NFT's and so on. As there are a lot of individuals that are not ready yet to even understand, the advanced technology. You can find the wallet according to the place you earned it from. Contact customer support if the place you earned Ethereum balance can't be found anywhere, to allow other people to transfer some of our their Ethereum that they earned or made a smart contract with. Kindly find example of a wallet account for a certain platform called Rain which you can trade Etherium with. Please take under consideration that the application rain is a wallet platform IE you can't daily trade and function deeply within finance and most are scammers. A trading platform on the other hand, like Binance, can be also used as a wallet but offers high profits with very high risks along with their CFD contracts they offer, margins, which we do not recommend at all. 

 but n . This is only an example of how a wallet can be found,  


 - Buy developers, brokers, different kinds of Websites that provides interaction with  

**How to create and find an Ethereum Wallet?**

We simply allow you to follow your true intentions, desires, dreams , and more by sharing :
 - Straight from the developers.
 - Trading Platforms.
 -Wallet Platforms.
 -Games.
 and much more.
 In summary:- Adding value is needed everywhere. Meanwhile creating it, is uncommon.
 Regards,
 
 Mohammed K. Al Saawi 
 Ethereum Community Coordinator

**Comment by samajammin** [comment]
Happy October!

We had an amazing time meeting many [ethereum.org](http://ethereum.org) contributors at [Devcon](https://devcon.org/) this month and encourage everyone to check out the [many talks](https://archive.devcon.org/archive/playlists/devcon-6/)! [Here’s my presentation on ethereum.org](https://archive.devcon.org/archive/watch/6/education-as-a-public-good-past-present-and-future-of-ethereumorg/?tab=YouTube) if you’re curious - I hope it provides useful context on this project & our vision for the future.

Closing this issue out with a progress update on each of our Q3 initiatives.

## 🔀 Post-Merge prep

The Ethereum network underwent [the largest upgrade in its history](https://ethereum.org/en/upgrades/merge/), fully transitioning to a proof-of-stake secured network. This brought important changes to how Ethereum functions, how it can scale, and the environmental impact from the elimination of proof-of-work.

**ethereum.org**

We audited all content on [ethereum.org](http://ethereum.org) to make sure our information is accurate and up-to-date in a post-Merge world for stakers, node operators, dapp developers and general users. With help from the community, we re-wrote many pages, including basic learning material, developer docs, and tutorials. We also added new pages, including Beacon Chain documentation from the [Knowledge Base at Beaconcha.in](https://kb.beaconcha.in/) (shoutout to Bitfly for the open source content!) and identified gaps to be addressed in the future with new pages.

See #7075 and corresponding PRs for a full list of updates.

**launchpad.ethereum.org**

We updated the Staking Launchpad with a “Merge Readiness Checklist” for all staking node operators on how to be prepared for The Merge. This included all necessary tasks that stakers must address in order to reduce the chances of downtime, penalties or slashing, and continued to emphasize the importance of client diversity, both for consensus clients as well as execution clients, with expanded links to client-specific setup instructions.

Now that the Ethereum community fully executed The Merge, with excellent participation rate from node operators and stakers, we deprecated the “Merge Readiness Checklist” and moved pertinent information into the [Validator checklist](https://launchpad.ethereum.org/en/checklist) and [FAQ](https://launchpad.ethereum.org/en/faq) pages, so new stakers can continue to access this important information.

See https://github.com/ethereum/staking-launchpad/issues/527 and corresponding PRs for a full list of updates.

Did we miss anything? Any new Merge-related resources we should create? Join the conversation in our [#merge Discord channel](https://discord.gg/aEgUGmEHUT).

## 🌏 Translate the EF blog

We launched the complete rebuild of the blog, which now supports 16 (and growing) languages! Learn more about the updates and future plans here:
[https://blog.ethereum.org/2022/08/31/blog-internationalization](https://blog.ethereum.org/2022/08/31/blog-internationalization)

Want to get involved with translations? Learn more:
[https://ethereum.org/en/contributing/translation-program/blog-translations/](https://ethereum.org/en/contributing/translation-program/blog-translations/)

Have any questions? Hop in our [#translate Discord channel](https://discord.gg/bTCfS8C) to chat with the team and other translators.

## 🎨 Open design system

This epic had reduced focus in Q3 as a result of Geth website design work (see Geth update below) but we still managed push progress forward across 1) foundations, 2) components, and 3) layouts. We synced the design system with our Chakra UI implementation, laying the groundwork for the upcoming code implementation of the design system.

See #6284 for a detailed breakdown of components we progressed on and usage stats. Expect us to continue this work through Q4 in coordination with the UI library rollout.

Want to get involved? [Follow along in Figma](https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System) and join the conversation in our [#design Discord channel](https://discord.gg/bKycYhVUwV).

## 🖥 Implement UI library

We began the migration of our UI components to Chakra and [created a migration guide](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md) to help community members contribute. Our first wave of 66 components is in progress - 28 components have been successfully migrated with an additional 17 open PRs are under review. We’re psyched to see contributions from 25 community members so far (including 22 new contributors)! We’ll be rolling out a second wave of components to migrate soon, so stay tuned!

See #6374 and corresponding PRs for a full list of updates.

Shoutout to community members @SNikhill, @Mousticke, @vdusart, @marcellamalune, @LuisUrrutia, @frankiefab100, @MahendraBishnoi29, @Master7130, @TylerAPfledderer, @derajohnson, @douglasmakey, @ameeetgaikwad, @ImMiguelP, @laurentlucian, @victorpatru, @AntonioSalazar, @AliTechGeek52, @Gift-Stack, @shariqanwar20, @shubham16598, @therealharpaljadeja, @hannsu, @Mahmadabid, and @jb-0 who contributed to this rollout!

Want to get involved? Join the conversation in our [#](https://discord.gg/7WKTFRARe5)[ui-library-migration Discord channel](https://discord.gg/TsKnq6nfZM).

## 🦔 Build a new Geth website

In collaboration with the Geth team, we’ve completely redesigned, restructured, rewrote and began rebuilding a new Geth website! We expanded documentation with several new pages and brought content up to date with the latest release of the go-ethereum client.

Take a sneak peak of our [designs](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=6%3A31) and [development progress](https://geth-website-git-homepage-mobile-ethdevops.vercel.app/). We’ll finish development and ship the new site in Q4.

See https://github.com/ethereum/geth-website/issues/1 and corresponding PRs for a full list of updates.

## 💖 Implement HEART framework

We analyzed 5 different aspects of the translator experience (Happiness, Engagement, Adoption, Retention, Task success) and collected stats that provide insights into each of these.

Based on these insights, we identified 65 concrete action items that will allow us to improve the contributor experience. Additionally, we have come up with a HEART framework retrospective with possible improvements to the process to potentially implement the framework across all contributors to ethereum.org

Expect us to start executing on these takeaways in Q4, so stay tuned for our next roadmap update!

Want to get involved? Join the conversation in [our #translate Discord channel](https://discord.gg/bTCfS8C).

## 💰 Wallet upgrades

We knocked out some cleanup work this quarter to remove unused data in the codebase for wallets and added [~10 new wallets](https://github.com/ethereum/ethereum-org-website/issues?q=is%3Aissue+label%3A%22wallet+%3Apurse%3A%22+is%3Aclosed) to give users more options, particularly for hardware wallets. Work will be carrying over into Q4, especially on UX improvements, which we've created issues for Hacktoberfest to give the community ways to contribute.

See #7250 and corresponding PRs for a full list of updates.

## 💯 Learning quizzes

The aim of creating learning quizzes is to ensure that readers are able to grasp the essential concepts of Ethereum to to test out their knowledge as they progress. We’ve nearly shipped an initial version! We tested quizzes on real-life users at Devcon and adapted the UX and quiz content based on early feedback.

See #7461 and the corresponding PRs for a full list of updates. Expect us to ship #8094 within the next week!

## 🗣️ User insights

Since rolling out the feedback widget (#6252), we’ve received thousands of survery responses from users and gathered lots of useful feedback from the community. You can view results from the feedback widget and surveys in our [Data Studio dashboards](https://datastudio.google.com/reporting/0a189a7c-a890-40db-a5c6-009db52c81c9/page/p_qfl0rcfbvc). Thank you all who have given us input!

Our Q4 roadmap and process to create new pages will be influenced by the insights gained from this feedback. We’ll continue to monitor user recommendations as well as start benchmarking individual page performance based on the received data.

Want to get involved? Join the conversation in our [#design Discord channel](https://discord.gg/bKycYhVUwV).

## 🌍 Revamp translation program resources

We rebuilt the [Translation Program page](https://ethereum.org/en/contributing/translation-program/) to improve the information architecture, added a summary section, translation pageview charts, translator acknowledgements overview, and listed projects that our community of contributors is translating. We created subpages on [How to translate](https://ethereum.org/en/contributing/translation-program/how-to-translate/) section, our [mission and vision](https://ethereum.org/en/contributing/translation-program/mission-and-vision/), [Launchpad translations](https://ethereum.org/en/contributing/translation-program/launchpad-translations/), [EF blog translations](https://ethereum.org/en/contributing/translation-program/blog-translations/), and [translator resources](https://ethereum.org/en/contributing/translation-program/resources/), in order better organize the information and make it easier to expand as the program grows in the future.

See #7206 and the corresponding PRs for a full list of updates.

Want to get involved? Join the conversation in [our #translate Discord channel](https://discord.gg/bTCfS8C).

# The nice-to-haves

_These epics are up next on our priority list. We’re not committing to shipping anything this quarter, but we plan to make progress on these and want to share these publicly so the community can give input and get involved!_

## 🏫 Ethereum for educators

We didn’t focus on this epic in Q3. While we did review all of the resources for relevancy and created more compact lists, this still needs design and development work to be completed.

See #6283 for details.

Want to get involved? Join the conversation in our [#content Discord channel](https://discord.gg/nZsD6hTxff).

## 📖 Content strategy

We didn’t create a content calendar this quarter but we made several valuable insights that will inform our content roadmap. Special shoutout to community member @abatrachenko for their contributions on keyword research and content briefings! In Q4, we will continue to work with them on an ad-hoc basis to identify content opportunities.

See #7240 for details.

Want to get involved? Join the conversation in our [#content Discord channel](https://discord.gg/nZsD6hTxff).

## 🪜 Restructure documentation

Unfortunately, outside of a few brainstorm sessions, we haven’t had time to make progress on this initiative.

See #7425 for details.

Want to get involved? Join the conversation in our [#content Discord channel](https://discord.gg/nZsD6hTxff)!

## 💰 Wallet resources

We researched some of the topics outlined for expanding resources, but we did not get to creating any content for expanding wallet resources this quarter due to other higher priorities.

See #7251 for details.

Want to get involved? Join the conversation in our [#content Discord channel](https://discord.gg/nZsD6hTxff).

## 🎡 JSON-RPC playground

Unfortunately we weren’t able to kick off this initiative this quarter. If you’re interested in helping out with this initiative, please let us know!

Want to get involved? Join the conversation in the [#json-rpc-api channel of the Eth R&D Discord](https://discord.gg/NJW4BJe3XZ).

# How does that sound?

We always appreciate feedback on our roadmap - if there's something you think we should work on, please let us know! We welcome ideas and PRs from anyone in the community.

Get involved: [Learn more about contributing](https://ethereum.org/en/contributing/), [hit us up on Twitter](https://twitter.com/ethdotorg), or join the community discussions in [our Discord](https://discord.gg/CetY6Y4).

**Comment by samajammin** [comment]
Updates on our Q3 KPIs:

**Translation KPIs**

- Reduce the number of QA issues in Crowdin by 20%
    - ❌ 6% reduction in QA issues since Q2
    - Still a 49.19% reduction since the first QA round in April!
- Update 200 content buckets between all languages
    - ✅ 276 content buckets updated
    - Kudos to all translators who helped us achieve this goal!
- Add translations for Essential pages in at least five new languages
    - ✅ 8 languages added
    - Chinese Traditional, Dutch, Filipino, Hindi, Korean, Malay, Swedish, Thai
- Add 5 new language leads
    - ❌ 4 language leads added
    - If you’re interested in becoming a language lead, please let us know!

**Community KPIs**

- Allow less than 2 PRs open longer than one month
    - ❌ 2 PRs open for >1 month at end of Q3
- Hit 15 GitHub power contributors (at least 3 PRs)
    - ✅ 16 power contributors in Q2
    - Huge thanks to everyone who contributed in any way!
- Add 5 new Discord guides
    - ❌ 0 Discord guides added
    - If you’re active on our Discord and think you’d make a good guide, please let us know!
- Host 3 community calls (monthly public calls)
    - ❌ 2 community calls hosted
- Host 3 community MVP hangouts (monthly get-togethers for language leads, Discord guides & GitHub power contributors)
    - ✅ 3 community MVP hangouts hosted
    - If you’d like to join an MVP hangout, ask us how you can get involved!

**Comment by KSST88** [comment]
#10140
